web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -287,7 +287,7 @@ export namespace MenuOverlay {
         return;
       }
 
-      if (event.code === Key.Escape || event.code === Key.Enter) {
+      if (event.code === Key.Escape) {
```

Comment:
I believe there are cases in which people want it to close when entered. So maybe we can add a boolean property in which when true, it will close on enter? Imagine a list and when you select an item in the list, the menuOverlay closes. Just thought we could add the flexibility to add that functionality back in if wanted.

Reply:
I second this

Reply:
Sure will incorporate the changes

Reply:
I looked into it. So currently what is happening is if you press Enter on the item in the list (according to the example you gave above) nothing will happen. It just closes. It will not select and then close. The bug was raised because of this only. Because on the desktop side people wanted the menu overlay to close upon pressing Enter on Submit but right now it is just closing and not performing the action on Submit. 
At every place in the desktop after the Enter they are closing the modal using ref. 
So according to me if we are giving Enter using the boolean property it will be useful only in the situation where user has opened menu overlay just to read the content and not perform any action.

Reply:
Ok